### ISSUE TITLE
The animation of the slider range circle will not work properly when the cursor returns to the range circle at the end of its gradual exit

### ISSUE THREAD

**Issue by PlayandyoU** [ppy/osu#22228]
### Type

Game behaviour

### Bug description

when the slider range circle is ending to fade out, the animation of the cursor returning to the range circle inside.
the range circle cannot work normally (the reduction when the range circle fades out) 
and when the slider is released, the range circle does not have the effect of enlarging and gradually emerging like the stable.

this bug find by player 13roil

### Screenshots or videos

https://youtu.be/CnWiuSGD0zk

https://user-images.githubusercontent.com/105141669/212600618-de571527-001c-49e3-8a5e-078083ab55c4.mp4

### Version

12.28

### Logs

[logs.zip](https://github.com/ppy/osu/files/10422569/logs.zip)

**Comment by peppy** [comment]
Duplicate of https://github.com/ppy/osu/issues/22150?

**Comment by PlayandyoU** [comment]
> 

Not exactly. This will happen if the track is not followed completely. lazer later than stable of the sliderend animation.
![image](https://user-images.githubusercontent.com/105141669/212601997-86eb3b43-9367-46e0-a37c-62f2408ae075.png)

**Comment by PlayandyoU** [comment]
> > 
> 
> Not exactly. This will happen if the track is not followed completely. lazer later than stable of the sliderend animation. ![image](https://user-images.githubusercontent.com/105141669/212601997-86eb3b43-9367-46e0-a37c-62f2408ae075.png)

and sliderend fate out time later than stable too

**Comment by peppy** [comment]
There's definitely some differences, but this feedback needs to be processed into a consumable format. The videos have discrepancies in gameplay which make them quite hard to follow.

**Comment by Detze** [comment]
Based on the video, the non-duplicate issue here is that on re-entering the follow area after leaving it after activating the slider, in lazer the Press animation is always played, while in stable it is not played if the leaving-and-re-entering did not result in a combo break.

This happens twice:
* at about 40% of the final repeat slide; interestingly, displayed cursor positions don't quite match between lazer and stable; it seems in stable the cursor stays in the follow area throughout, while in lazer the cursor leaves and re-enters the follow area, strange;
* at about 90% of the final repeat slide: this seems to extend the follow circle fade out animation, presumably due to the Press animation finishing first (the slider was reactivated in the tail leniency period, [related issue](https://github.com/ppy/osu/issues/25981), I believe this unnecessary lengthening would be fixed by [my PR](https://github.com/ppy/osu/pull/27352)), making it stay on the screen for longer. It also shows how [playing Break at `end_time - 36` instead of `end_time` in lazer](https://github.com/ppy/osu/issues/22150#issuecomment-1963014309) is confusing, as the follow circle briefly becomes larger while entering the tail leniency region.
